src/database/laneTimebandDb.ts

```diff
@@ -0,0 +1,83 @@
+import logger from '../util/logger';
+import { dbConnection } from './dbConnection';
+import { LaneTimeband } from '../interfaces/LaneTimeband';
+import { VtBooking } from '../interfaces/VtBooking';
+
+export const laneTimebandDb = {
+  async insert(vtBooking: VtBooking): Promise<void> {
+    logger.info('laneTimeband insert starting');
+    const connection = await dbConnection();
+
+    const laneTimeband = {
+      ...new LaneTimeband(),
+      FK_STATN_ID: vtBooking.pNumber,
+      TIMESTAMP0: connection.raw('sysdate'),
+      DATE0: connection.raw(`to_date('${vtBooking.testDate}', 'yyyy-mm-dd')`),
+      OPEN_TIME: connection.raw(
+        `to_date('${vtBooking.timeband.openTime}', 'yyyy-mm-dd hh24:mi:ss')`,
```

Comment:
Niggle, but these date formats across a few of these files could be specified once in a `utils` or enum file

```
const enum DatabaseDateFormats {
  date = 'yyyy-mm-dd',
  dateTime = 'yyyy-mm-dd hh24:mi:ss'
}
```

```suggestion
      DATE0: connection.raw(`to_date('${vtBooking.testDate}', '${DatabaseDateFormats.date}')`),
      OPEN_TIME: connection.raw(
        `to_date('${vtBooking.timeband.openTime}', '${DatabaseDateFormats.dateTime}')`,
```

Reply:
That's a good suggestion. I'll add it on the next ticket.